torch/csrc/jit/register_quantized_ops.cpp

```diff
@@ -28,7 +28,7 @@ using caffe2::int8::Int8TensorCPU;
 namespace {
 
 caffe2::Tensor from_at_tensor(const c10::IValue& v) {
-  return caffe2::Tensor(autograd::Variable(std::move(v).toTensor()).data());
+  return caffe2::Tensor(std::move(v).toTensor());
```

Comment:
@dzhulgakov Does it matter if the `caffe2::Tensor` returned here can potentially contain AutogradMeta? The downside of having AutogradMeta in the caffe2::Tensor's TensorImpl is that the memory usage will be slightly larger. We can also try to make a shallow copy of the `at::Tensor` here to get rid of the AutogradMeta, if the extra memory usage is an issue. Curious what do you think would be the right approach here?

Reply:
Based on discussion with Dima, this is more of a stub for prototype, and we can delete `.data()` here.

Reply:
don't we have an enforce invariants that @dzhulgakov wrote where we could just put that enforce for now?  We probably want to get rid of it later, but seems reasonable to start with the most restrictive thing.

Reply:
I found that we need to allow passing with-autograd-meta tensors into Caffe2, with reasons at https://github.com/pytorch/pytorch/pull/17072/files#r281771685.